### PR TITLE
For #8500: Show size in KB for smaller downloaded files.

### PR DESCRIPTION
**Pull request by mcarare**
---

### Pull Request checklist

- [ ] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/android-components/blob/master/docs/changelog.md) or does not need one
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features

### After merge
- **Milestone**: Make sure issues closed by this pull request are added to the [milestone](https://github.com/mozilla-mobile/android-components/milestones) of the version currently in development.
- **Breaking Changes**: If this is a breaking change, please push a draft PR on [Reference Browser](https://github.com/mozilla-mobile/reference-browser) to address the breaking issues.

### CODE REVIEW

File: components/feature/downloads/src/main/java/mozilla/components/feature/downloads/DownloadDialogFragment.kt [hunk 1]
```diff
@@ -65,3 +73,22 @@ abstract class DownloadDialogFragment : AppCompatDialogFragment() {
...
+fun Long.toKilobyteString(): String {
+    return String.format("%.2f KB", this / KILOBYTE)
+}
+
+/**
+ * Converts the bytes to megabytes or kilobytes( if size smaller than 0.01 MB)
+ * with two decimal places and returns a formatted string
+ */
+fun Long.toMegaByteOrKilobyteString(): String {
+    return if (this / MEGABYTE < BYTES_TO_MB_LIMIT) {
```
Comment (rocketsroger) [inline components/feature/downloads/src/main/java/mozilla/components/feature/downloads/DownloadDialogFragment.kt:89]: Can you please add a bracket around (this / MEGABYTE)? From the operator precedence table it looks like the `<` operator is higher precedence.

**Review by rocketsroger — COMMENTED** [review]
Thanks! Can you please add tests for this change?

File: components/feature/downloads/src/main/java/mozilla/components/feature/downloads/DownloadDialogFragment.kt (hunk 1 above)
Comment (mcarare) [reply components/feature/downloads/src/main/java/mozilla/components/feature/downloads/DownloadDialogFragment.kt:89]: Multiplicative operators have higher precedence than Comparison operators.

Comment (rocketsroger) [reply components/feature/downloads/src/main/java/mozilla/components/feature/downloads/DownloadDialogFragment.kt:89]: Ah my mistake. 😓   Thanks

**Review by rocketsroger — APPROVED** [review]
🚢 Thanks!

**Pull request merged**